includes/Story_Post_Type.php

```diff
@@ -414,6 +420,113 @@ public function filter_use_block_editor_for_post_type( $use_block_editor, $post_
 		return $use_block_editor;
 	}
 
+	/**
+	 * Returns list of custom meta boxes per location.
+	 *
+	 * Used to disable empty meta boxes in the editor.
+	 *
+	 * @see the_block_editor_meta_boxes()
+	 *
+	 * @since 1.2.0
+	 *
+	 * @return array List of meta boxes per location.
+	 */
+	public function get_meta_boxes_per_location() {
```

Comment:
This class is really very long. Should this logic live in a metabox class of it's own? 
```
$metabox = new Metaboxes( Story_Post_Type::POST_TYPE_SLUG );
$metabox->init();
```

 Or an abstract class called `Post_Type`, that both `Story_Post_Type` and `Template_Post_Type` extend. That way we can keep much of the shared logic between post types shared.

Reply:
> This class is really very long. Should this logic live in a metabox class of its own?

I'll take a stab at that.
 
> Or an abstract class called `Post_Type`, that both `Story_Post_Type` and `Template_Post_Type` extend. That way we can keep much of the shared logic between post types shared.

That's something that's better explored & discussed at #5062

Reply:
There is no need to make this class longer and make the problem worse 😛